app/linter/ink-linter.js

```diff
@@ -57,8 +124,28 @@ let storyTags = {
   linter: hasStoryTagError,
   needsParam: true,
   validParams: {
-    cutCamera: ['linapov', 'paxleft', 'paxright', 'dashcam', 'hoodcam', 'linafront', 'city'],
-    linaHands: ['checkwatch']
+    cutCamera: [
+      'linapov',
+      'LinaFrontClose',
```

Comment:
Should these be all lower for the linter?

Reply:
@awmcclain  - fixed!